README.md

```diff
@@ -57,3 +57,7 @@ Antenna can be used with Java versions 9 or newer.
 However, it requires some additional configuration described in [Tool Configuration](antenna-documentation/src/site/markdown/tool-configuration.md.vm/#additional-configuration-for-java-9-or-newer).
 
  *To find answers in the most frequent questions/problems go to [Troubleshooting](antenna-documentation/src/site/markdown/troubleshooting.md.vm).*
+
+### Documentation
+
+For more information please consider our [documentation](https://eclipse.github.io/antenna/1.0.0-SNAPSHOT/).
```

Comment:
s/consider/refer to/?

Can we somehow omit the hard-coded version from the link so that we don't have to update the link with new releases? Like by introducing a "latest" symlink?